packages/loopring_v3/contracts/impl/ExchangeV3.sol

```diff
@@ -531,26 +442,17 @@ contract ExchangeV3 is IExchangeV3
     }
 
     function deposit(
-        address token,
-        uint96  amount
-        )
-        external
-        payable
-        nonReentrant
-    {
-        state.depositTo(msg.sender, token, amount, 0);
-    }
-
-    function depositTo(
-        address recipient,
+        address from,
```

Comment:
I think using `owner` for `from` here is a bit confusing. It's the `from` address that transfers funds to the account of the `to` address. So actually it's the `to` address that's the owner of the account that will receive the funds, `from` is just the source of the funds. So using `owner` for one of them seems more confusing to me than using `from` and `to` because `from` is the owner of the funds being transferred and `to` is the owner of the account that will receive the funds. Phew, what an explanation. :)

In short, because the account stuff makes things complicated I named things as though there are no accounts and it's a simple onchain operation that just transfers funds from an address to another address.